tests/test_factories/test_application_round.py

```diff
@@ -13,3 +15,4 @@
 def test_application_round_factory_create_in_status(status):
     application = ApplicationRoundFactory.create_in_status(status=status)
     assert application.status == status
+    assert Application.objects.filter(L(status=status)).exists()
```

Comment:
This is an ApplicationRound status check test, but the round is saved to `application` variable 😄 Who could have done that... Anyway, shouldn't check `Application.objects` here, but `ApplicationRound.objects`.

Reply:
Nice catch! Fixed :)